public/js/userVerification.js

```diff
@@ -0,0 +1,29 @@
+let acceptBtns = document.querySelectorAll('.user-verification-btn--accept');
+let verifContainer = document.querySelector('.user-verification-wrapper');
+
+for (let acceptBtn of acceptBtns) {
+    acceptBtn.addEventListener('click', function () {
+        console.log(acceptBtn.parentElement.dataset.userId);
+        updateVerification(acceptBtn.parentElement.dataset.userId);
+        verifContainer.style.display = "none";
+    })
+}
+
+let rejectBtns = document.querySelectorAll('.user-verification-btn--reject');
+let rejectBtnContainer = document.querySelector('.user-verification-wrapper');
```

Comment:
What does line 13 for?

Reply:
oh sorry, forgot to remove it, let me commit and push again